indigo_app/templates/indigo_api/work_overview.html

```diff
@@ -67,6 +67,17 @@ <h4 class="text-muted">{{ work.frbr_uri }}</h4>
                     <br>Commenced by <a href="{% url 'work' frbr_uri=work.commencing_work.frbr_uri %}" data-popup-url="{% url 'work_popup' frbr_uri=work.commencing_work.frbr_uri %}">{{ work.commencing_work.title }}</a>
                     {% endif %}
                   </dd>
+
+                  {% regroup work.taxonomies.all by taxonomy_vocabulary as topic_groups %}
+                  {% if topic_groups %}
+                      {% for topic in topic_groups %}
+                        <dt>{{ topic.grouper }}</dt>
+                          {% for taxonomy in topic.list %}
```

Comment:
Inside a `dl` element, there should only be `dt` and `dd` tags. So wrap the for loop in a `dd` tag.

Reply:
i have updated the file with the suggestions, however it looks like the line spaces are a bit big between the `dd` tags